src/connecpy/client.py

```diff
@@ -76,31 +77,28 @@ def _make_request(
                     url=self._address + url, content=request_data, **kwargs
                 )
 
-            resp.raise_for_status()
-
             if resp.status_code == 200:
                 response = response_class()
                 try:
                     response.ParseFromString(resp.content)
-                    return response
                 except Exception as e:
                     raise exceptions.ConnecpyException(
                         f"Failed to parse response message: {str(e)}"
                     )
+                return response
             else:
-                raise exceptions.ConnecpyServerException.from_json(resp.json())
+                raise ConnectWireError.from_dict(
+                    resp.json(), resp.status_code
```

Comment:
Should we handle cases where `resp.json()` fails? (I did nothing, though..)

Reply:
Oh thanks! I reworked to handle it and added tests for it and any other non-connect HTTP response